mne/time_frequency/psd.py

```diff
@@ -147,12 +342,11 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
         Number of CPUs to use in the computation.
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
-
```

Comment:
yeah sorry about that - I was reverting code back to previous versions and did it sloppily